addons/web/static/src/js/components/dropdown_menu.js

```diff
@@ -52,13 +52,18 @@ odoo.define('web.DropdownMenu', function (require) {
         }
 
         /**
-         * In mobile, by default, we display a chevron icon next to the dropdown
-         * button. Note that when 'displayCaret' is true, we display a caret
-         * instead of a chevron, no matter the value of 'displayChevron'.
          * @returns {boolean}
          */
         get displayChevron() {
-            return this.env.device.isMobile;
+            return false;
+        }
+
+        /**
+         * By default, we only display the title in desktop
+         * @return boolean
+         */
+        get displayTitle() {
```

Comment:
:warning: This is dangerous as it applies on *all* DropdownMenu (like in the search view)